source/includes/_capture_external_event_api.md

```diff
@@ -0,0 +1,72 @@
+# Capture External Event API
+
+Capture External Event API is a public ReachLocal external interface for vendors to use.
+
+## Authorization - Capture External Event API
+
+All requests must include an [Authorization](#authentication) header.
+
+The email address used to obtain the OAuth access token must be associated with the GMAID of the requested advertiser.  If it is not, the API will return an HTTP response with status code 403 and the following body:
+
+`{ "message": "Forbidden" }`
+
+## Create Chat Event
+
+This endpoint is used to create a chat event.
+
+### HTTP Request
+
+| Method | URI Format |
+|---|---|
+| POST | https://externalapi.reachlocal.com/api/chats |
+
+### Post Body Parameters
+
+Parameter | Required | Datatype | Nullable | Description
+--------- | -------- | -------- | -------- | -----------
+eventTime | yes  | string | no | The date/time in UTC that the chat started.
+advertiserId | yes*  | string | no | The global master advertiser id (for example, USA_142687).
+scopeId | yes*  | string | no | Legacy use only. The global master advertiser id (for example, USA_142687).
```

Comment:
If we don't want users to call us using this parameter, why document it as legacy?  Our code can continue to support this parameter for backwards compatibility, but there might not be a reason to include it in the documentation.

Reply:
Yeah, this one is causing me headaches all around. I am going to check with Eric to see if we can remove it.